mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -218,31 +217,51 @@ class LogisticRegression(override val uid: String)
 
   override def getThreshold: Double = super.getThreshold
 
+  /**
+   * Whether to over-/undersamples each of training sample according to the given
```

Comment:
* minor: `instance` might be better than `sample` here. `sample` might mean a sampled subset.
* `each of training sample` -> `each training instance`